catroid/src/main/java/org/catrobat/catroid/utils/Utils.java

```diff
@@ -479,6 +476,15 @@ public static double round(double value, int precision) {
 		return (double) Math.round(value * scale) / scale;
 	}
 
+	private static void displayDialog(Context context, final Builder builder) {
+		((Activity) context).runOnUiThread(new Runnable() {
```

Comment:
The cast is just a quick-fix and will throw an exception if the Context can't be casted to `Activity`. Please fix appropriately.